pkg/proxy/proxy_start.go

```diff
@@ -60,6 +61,10 @@ func (p *Proxy) Start() {
 		ls := ListenerService{Listener: &s}
 		go ls.Start()
 	}
+
+	if conf.GetPprof().Enable {
+		go http.ListenAndServe(conf.GetPprof().Addr, nil)
```

Comment:
I think maybe pprof goruntine can stop by command, it's  dync, so need some api to control it.